src/pages/FridgeDetailPage.js

```diff
@@ -1 +1,223 @@
-export const FridgeDetailPage = () => null;
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Text, View } from 'react-native';
+import * as Animatable from 'react-native-animatable';
+
+import { DateRangeSelector } from '../widgets/DateRangeSelector';
+import {
+  CardDetail,
+  ColdBreachIcon,
+  DataTablePageView,
+  FlexRow,
+  HotBreachIcon,
+  PaperSection,
+  SpacedChildren,
+} from '../widgets';
+import { VaccineChart } from '../widgets/VaccineChart';
+import { NoBreachMan } from '../widgets/NoBreachMan';
+import { FridgeActions } from '../actions/FridgeActions';
+import { AfterInteractions } from '../widgets/AfterInteractions';
+
+import {
+  selectAverageTemperature,
+  selectBreaches,
+  selectColdCumulativeBreach,
+  selectHotCumulativeBreach,
+  selectLeastRecentTemperatureLogDate,
+  selectMinAndMaxDomains,
+  selectMinAndMaxLogs,
+  selectMostRecentTemperatureLogDate,
+  selectNumberOfColdConsecutiveBreaches,
+  selectNumberOfHotConsecutiveBreaches,
+  selectTemperatureLogsFromDate,
+  selectTemperatureLogsToDate,
+} from '../selectors/fridge';
+
+import {
+  DARKER_GREY,
+  APP_FONT_FAMILY,
+  BLUE_WHITE,
+  COLD_BREACH_BLUE,
+  DANGER_RED,
+} from '../globalStyles';
+
+const CardText = ({ children, color, small }) => (
+  <Text style={{ color, fontSize: small ? 12 : 50, fontFamily: APP_FONT_FAMILY }}>{children}</Text>
+);
+
+CardText.defaultProps = { small: false };
+
+CardText.propTypes = {
+  children: PropTypes.node.isRequired,
+  color: PropTypes.string.isRequired,
+  small: PropTypes.bool,
+};
+
+const BreachCard = ({ content, type }) => {
+  const color = type === 'HOT_CUMULATIVE' ? DANGER_RED : COLD_BREACH_BLUE;
+  const Icon = type === 'HOT_CUMULATIVE' ? HotBreachIcon : ColdBreachIcon;
+
+  const header = type.includes('CUMULATIVE') ? 'Cumulative Breach' : 'Consecutive Breach';
+
+  return (
+    <CardDetail
+      headerText={header}
+      Content={content ? <CardText color={color}>{content}</CardText> : <NoBreachMan />}
+      Footer={content ? <Icon color={color} /> : <CardText small>No breaches!</CardText>}
+    />
+  );
+};
```

Comment:
Don't usually like having multiple components in the same file but also couldn't really justify taking them out.. I dunno